docs/lib/usage/starlite-app/app-hooks.rst

```diff
@@ -0,0 +1,166 @@
+ASGI Events & Application Hooks
+===============================
+
+In the previous introductory section of the documentations we briefly mentioned about the root
+:class:`Starlite <starlite.app.Starlite>` class. This class accepts a bunch of `optional` arguments
+& a couple of those arguments can be configured to control the behaviour of your Starlite application
```

Comment:
please don't use `&` at all. This is a reapting issue. 

I think we also agreed on using American spelling. Hence -> `behaviour` -> `behavior` etc.

Reply:
Yeah, my bad about using `&`, I keep using it out of habit :sweat_smile:. I will do a search & replace before marking this PR as "ready for review".